### ISSUE TITLE
Merge hu (Hungarian - magyar) locale translations from Transifex

### ISSUE THREAD

**Issue by ovari** [ExchangeCalendar/exchangecalendar#234]
Please merge the hu (Hungarian - magyar) translation from [Transifex](https://www.transifex.com/ExchangeCalendar/exchangecalendar/). Thank you

**Comment by ovari** [comment]
Sorry @advancingu, don't know how to test.

**Comment by advancingu** [comment]
Ok, try this build (unzip first):
[exchangecalendar-v5.0.0-alpha2.zip](https://github.com/ExchangeCalendar/exchangecalendar/files/2751400/exchangecalendar-v5.0.0-alpha2.zip)

**Comment by ovari** [comment]
`install.rdf.template`

```
        <em:localized>
            <Description>
                <em:locale>hu</em:locale> <!-- Hungarian -->
                <em:name>Exchange-naptár (Exchange Calendar)</em:name>
                <em:description>A naptárak, a feladatok és a névjegyzék összehangolása a Microsoft Exchange szolgáltatással.</em:description>
            </Description>
</em:localized>
```

**Comment by ovari** [comment]
In the image below, need to update
Server URL: → Kiszolgáló URL:

![exchangecalendar](https://user-images.githubusercontent.com/17465872/51069853-39f2f600-168b-11e9-8abd-c37c590b2b08.png)

**Comment by ovari** [comment]
In the image below:
1. Window Title is in English, not Hungarian.
2. Window Title should not have a full stop.

What do you think?

Thank you

![image](https://user-images.githubusercontent.com/17465872/51071540-a0394200-16a6-11e9-9702-656ae93dd890.png)

**Comment by advancingu** [comment]
Merged into v5.0.0-alpha3.

**Comment by ovari** [comment]
Thank you
